src/main/java/com/zendesk/maxwell/schema/SchemaStoreSchema.java

```diff
@@ -99,6 +99,10 @@ public static void upgradeSchemaStoreSchema(Connection c) throws SQLException, I
 			performAlter(c, "alter table `schemas` add column deleted tinyint(1) not null default 0");
 		}
 
+		if ( !getTableColumns("schemas", c).containsKey("gtid_set") ) {
+			performAlter(c, "alter table `schemas` add column gtid_set varchar(255)");
```

Comment:
i think we should lengthen these out as much as possible.  Maybe even use a `TEXT` here?  If there's no reason against it, might as well future-proof....